jwst/photom/ifu_mrs.py

```diff
@@ -0,0 +1,145 @@
+import logging
+import numpy as np
+from scipy.interpolate import RegularGridInterpolator
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def expfunc_bounded(x, a, b, c, x0):
+    """ Function using time dependent coefficients """
+
+    temp = a * np.exp(-b * (x - x0) / 100) + c
```

Comment:
Curious what the units of `x` and `x0` are and hence the need for the division by 100. Perhaps add a list of parameter definitions giving that kind of info.